openroad/BUILD

```diff
@@ -27,11 +27,14 @@ py_package(
 
 py_wheel(
     name = "openroad_wheel",
+    abi = "abi3",
     distribution = "openroad",
     homepage = "https://github.com/The-OpenROAD-Project/OpenROAD",
-    python_tag = "py3",
+    python_requires = ">=3.10",
+    python_tag = "cp310",
```

Comment:
does this imply that only python3.10 will be supported?
It would be nice if the wheel could be built for 3.10, 3.11, 3.12, 3.13, and 3.14, otherwise this will not be particularly portable.
With the `openroad -python` we controlled the python version, but that is not necessarily true with something published on pypi.

Reply:
@QuantamHD 
Did you try with older versions, or was 3.10 what you had available?
Would it be worth trying with older versions to increase support?

Reply:
Python is a marginal use case in its current form compared to the new world of pypi....

Perhaps dropping Python from older RHELs is acceptable while still building with Bazel?